server/server.go

```diff
@@ -80,24 +80,24 @@ func build(w http.ResponseWriter, r *http.Request) {
 	var err error
 
 	// Read the command to compile.
-	assignTestingInfo.CommandToCompile = r.FormValue(constants.CompileCmdKey)
+	assignTestingInfo.CompileCommand = r.FormValue(constants.CompileCmdKey)
 
 	// Navigate to the assignment working directory.
 	currDir, err = navigateToWorkDir()
 	if err != nil {
 		outputString = err.Error()
 	} else {
 		// Execute the compile command.
-		outputString, err = runCommand(assignTestingInfo.CommandToCompile)
+		outputString, err = runCommand(assignTestingInfo.CompileCommand)
 		if err != nil {
 			log.Println("error while building the assignment", err)
 			outputString += err.Error()
 		}
 
 		// Navigate back to the code-runner working directory after successful execution.
-		errChdir := os.Chdir(currDir)
-		if errChdir != nil {
-			log.Println("error while navigating to the current directory", errChdir)
+		chdirErr := os.Chdir(currDir)
+		if chdirErr != nil {
+			log.Println("error while navigating to the current directory", chdirErr)
```

Comment:
inconsistency between error message and associated comment.

Reply:
Fixed